blog-posts/sls-security-headers/sls-security-headers.md

```diff
@@ -0,0 +1,96 @@
+---
+published: false
+title: 'Leverage AWS CloudFront managed response headers policies'
+cover_image: 'https://dev-to-uploads.s3.amazonaws.com/uploads/articles/xgzowe15czab4awhanxd.png'
+description: ''
+tags: cloudfront, security, AWS, audit
+canonical_url:
+---
+
+– Did you just say security headers?
+
+– Yes.
+
+– Are you talking about CORS headers?
+
+– No.
+
+# HTTP security headers: a very simple addition to your website to efficiently prevent attacks
+
+I have learnt about HTTP security headers very late in my full stack developer carrier. In the last 5 years, have worked in a dozen teams to develop and run web applications.
+
+This is only very recently, when I joined [Theodo](https://www.theodo.fr/), that I was explained what HTTP security headers actually are. 😮
+
+HTTP security headers are seemingly unknown to many full stack developers.
+
+HTTP security headers are a mechanism to **protect your application from common attacks**. HTTP security headers are, as their name suggests, a set of HTTP headers with security purposes.
+
+The table below shows a subset of HTTP security headers. The value is strongly recommended. The description mentions the common attacks that the header protects against.
+
+| Name | Value | Description |
+| --- | --- | --- |
+| [Strict-Transport-Security](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security) | max-age=31536000 | Informs browsers that the site should only be accessed using HTTPS, and that any future attempts to access it using HTTP should automatically be converted to HTTPS.
+The user's inputs will never be readable by a man on the middle (except potentially the first call ever, which does not contain any sensitive information). |
```

Comment:
You should remove this new line, the display is a bit off: 
![image](https://user-images.githubusercontent.com/19605940/214248686-e593ff7d-44dd-44cf-815d-9204e56c3650.png)